services/gafaelfawr/values-minikube.yaml

```diff
@@ -0,0 +1,20 @@
+gafaelfawr:
+  host: minikube.lsst.codes
+  ingress:
+    host: minikube.lsst.codes
+  vault_secrets_path: "secret/k8s_operator/minikube.lsst.codes/gafaelfawr"
+
+  # Session length and token expiration (in minutes).
+  issuer:
+    exp_minutes: 43200  # 30 days
+
+  # Disabled but kept so that the client ID is easily accessible.
+  # github:
+  #   client_id: a19e79298a352f3e5650
+
```

Comment:
Should delete this -- this was specific to whatever environment the file was cloned from.

Reply:
Well, yes and no.  The _idea_ of keeping the client ID around in the file is not a bad one, as long as we don't keep the secret in the file.  But it should definitely be *our* client ID if we keep it.